packages/uiScripts/src/scripts/contactUs/ContactUs.component.js

```diff
@@ -101,7 +101,8 @@ const ContactUs = () => {
       <Styled.PopoverContainer>
         <NavLinks />
         <Styled.GuideContent isSmallScreen={bp.xs}>
-          <p>遇到紧急事故、或是需要技术交流、社区合作可以在这里联系社区专家哦~</p>
+          {/* 遇到或是紧急事故、 */}
+          <p>需要技术交流、社区合作可以在这里联系社区专家哦~</p>
```

Comment:
Better to comment out the previous paragraph tag and add a new tag below. Then, when we'd like add the incident report back, we just remove the new paragraph tag and uncomment the previous tag.

```
{/* <p>遇到紧急事故、或是需要技术交流、社区合作可以在这里联系社区专家哦~</p> */}
<p>需要技术交流、社区合作可以在这里联系社区专家哦~</p>
```